config/locales/service_providers/en.yml

```diff
@@ -11,15 +11,3 @@ en:
           still active and can be used to access other participating government
           websites.'
         instructions2: Please visit the agency’s website to contact them for more information.
-    help_text:
```

Comment:
This was a bit harder to pin down, but as best as I can tell, it is unused.

Help text is referenced here, directly from the record:

https://github.com/18F/identity-idp/blob/37e8d8b9d31ade5258d35b736970f32338a2d74c/app/decorators/service_provider_session_decorator.rb#L25

I couldn't find any code which referenced this locale data directly.

Reply:
Looks it was added in #3320 and removed in #3336, so it should be safe?

Reply:
> Looks it was added in #3320 and removed in #3336, so it should be safe?

Thanks for digging those up! Makes sense to me.

(Aside: Further evidence that this configuration value tends to cause maintainability headaches 😅 )

Reply:
100% agree, we should be using `i18n-tasks-use` if strictly needed and otherwise avoiding dynamic/wildcard translation.